assignments/assignment_2/README.md

```diff
@@ -59,14 +59,14 @@ We know that the things presented in the previous modules are orientative guidel
 This exercise is very simple, we just want you to fish out some current or older project that you are or were part of and review its organization. What would you change? What would you keep? Why? 
 
 
-## 2- Structure recognition
+## 2- Structure analysis
 
-In collecting a couple of examples for the project structure module we made a mess and lost track of the structure each of the projects. Give it a go and see if you can figure out which patterns do these projects follow.
+Again, a simple exercise. The idea this time is to practise your ability to navigate projects you have not written yourself. 
```

Comment:
```suggestion
Let's practice your ability to navigate projects you have not written yourself. 
```

It's a good practice to remove words like "simple", "trivial", etc while writing tutorials or documentation. It's very frustrating when we say something is simple when the user finds it to be anything but.